source/renderer/app/api/api.js

```diff
@@ -554,6 +554,11 @@ export default class AdaApi {
         // @API TODO - Change error message when fee calculation fails regarding to not enough fragmented UTXO
         //           - Also check if error.code is correct
         throw new NotEnoughFundsForTransactionError();
+      } else if (
+        error.code === 'bad_request' &&
+        includes(error.message, 'Unable to decode Address')
+      ) {
+        throw new Error('INVALID_ADDRESS');
```

Comment:
@tomothespian don't we have a custom error message for this case?
Can this one even happen?

Reply:
We have just declaration in component and I separated that and handle where other component based errors are handled.
That can happen in case where we calculate transaction fee. In that case we will get `Error occured` soo I decided to show proper error

Reply:
Can we rather create a error declaration instead of `Error('INVALID_ADDRESS)`? Something like this? https://github.com/input-output-hk/daedalus/blob/develop/source/renderer/app/api/transactions/errors.js#L90

Reply:
Deal